packages/api/server/models/shantytownCommentModel/findOne.js

```diff
@@ -6,7 +6,22 @@ const { serializeComment } = require('#server/models/shantytownModel');
  */
 module.exports = async (id) => {
     const rows = await sequelize.query(
-        `SELECT
+        `WITH comment_tags AS (
+            SELECT
+                  sct.fk_shantytown_comment,
+                  ARRAY_AGG(sct.fk_comment_tag) AS tags
+              FROM
+                  shantytown_comments AS sc
+              LEFT JOIN
+                  shantytown_comment_tags AS sct
+              ON
+                  sct.fk_shantytown_comment = sc.shantytown_comment_id
```

Comment:
Par soucis d'homogénéité entre nos requêtes SQL, passer tout le left join sur une seule ligne